src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java

```diff
@@ -86,6 +86,10 @@ public void recommend(KruizeObject kruizeObject) {
                         continue;
                     Timestamp monitorEndTimestamp = containerData.getResults().keySet().stream().max(Timestamp::compareTo).get();
```

Comment:
Can you please add a comment that this assumes that the data is in-order. Ideally we need to pass in the `results` object that has currently been POSTED and use the `interval_end_time` that is part of that object as the `monitorEndTimestamp`.